KoansAndKatas/koanwiki.md

```diff
@@ -0,0 +1,59 @@
+###Intro
+We're exploring more ways to help the team get a stronger foundation of F#.  
+
+###Goals
+There are some goals I would like to achieve, all of which are part of the greater vision of empowering others, which is a core value of RealtyShares:  
```

Comment:
Replace all references of "I" with "we", to emphasize this is a team effort, not a personal ambition.